test/e2e/storage/mounted_volume_resize.go

```diff
@@ -163,7 +158,7 @@ var _ = utils.SIGDescribe("Mounted volume expand [Feature:StorageProvider]", fun
 		framework.ExpectNoError(err, "while deleting pod for resizing")
 
 		ginkgo.By("Waiting for deployment to create new pod")
-		pod, err = waitForDeploymentToRecreatePod(c, deployment)
+		pod, err = testsuites.waitForDeploymentToRecreatePod(c, deployment)
```

Comment:
Hi @Kartik494 this was not quite the ask in the issue. The ask is to adapt this test case into the [resize testsuite](https://github.com/kubernetes/kubernetes/blob/da56595c68f8320a181a031b631c172e555d1b27/test/e2e/storage/testsuites/volume_expand.go) so that it could be run against any storage driver. This will require moving the entire test case and a bit more changes around how the storageclass gets initialized

Reply:
@msau42  Thanks for the clarification,i will update it accordingly

Reply:
Yeah that sounds right. We wanted to move this test to common test suite.

Reply:
@gnufied Sure, i will modify this as per requirements.